### DISCUSSION TITLE
哔哩开发方向调整

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1174] [想法]
我将这段时间的思考和总结写在了 [哔哩项目概述](https://github.com/Richasy/Bili.Uwp/wiki/%E5%93%94%E5%93%A9%E9%A1%B9%E7%9B%AE%E6%A6%82%E8%BF%B0) 中。不过文章有点长，不想看的可以直接往下看缩略版：

简而言之，我认为按照现有路线企图在功能上尽善尽美，追平和官方应用的差距是不现实的，且没什么意义。哔哩需要转变开发思路，调整应用定位来面对未来的挑战。

所以我不准备再在用户体验和应用功能上继续投注精力，现在的回报（情绪价值和技术提升）已经极其微小了。哔哩的后续开发准备按照公私两个方向走。

- 于私：代码重构，实验学习到的编程范式、设计模式，提高自己的架构水平。将哔哩这个项目在代码层面做到优秀，吸引更多的开发者参与。

- 于公：探索如何组织BiliBili的信息，并通过各种方式将信息打散，融入到你的工作流之中，在信息茧房上挖个洞

**Comment by TimmonSha** [comment]
开发者很有想法，棒！

**Comment by Pigletzzz** [comment]
加油！

**Comment by aoout** [comment]
我曾经尝试过使用rss来订阅bilibili上的视频，过程中有很多难得的体验。不过因为rss软件内无法播放视频，每次都要跳转过于麻烦，已经放弃了。
